receiver/sqlserverreceiver/scraper.go

```diff
@@ -526,31 +528,31 @@ func (s *sqlServerScraperHelper) recordDatabaseQueryTextAndPlan(ctx context.Cont
 	}
 	var errs []error
 
-	totalElapsedTimeDiffs := make([]int64, len(rows))
+	totalElapsedTimeDiffsMilliSecond := make([]int64, len(rows))
 
 	for i, row := range rows {
 		queryHashVal := hex.EncodeToString([]byte(row[queryHash]))
 		queryPlanHashVal := hex.EncodeToString([]byte(row[queryPlanHash]))
 
-		elapsedTime, err := strconv.ParseInt(row[totalElapsedTime], 10, 64)
+		elapsedTimeMicroSecond, err := strconv.ParseInt(row[totalElapsedTimeSecond], 10, 64)
```

Comment:
Why use `totalElapsedTimeSecond` as the variable name? I think it is just a column name.

And, both `total_elapsed_time` and `total_worker_time` use microseconds in SQL Server. Why do they have different naming styles?

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/39027#discussion_r2018866700

this change is based on the above comment, which i think make sense. we have so many conversion in the code regarding the time and it would cause confusing when other people come and update the code. 

i updated total worker time variable name to reflect microsecond